### PR TITLE
Add all testimonials from 2021.06.25 to 2021.07.14

### PR DESCRIPTION
**Pull request by renderf0x**
### How to test

So long as tweet embeds load on the about page, the JSON is good to go.

### CODE REVIEW

File: data/testimonials.json [hunk 1]
```diff
@@ -22,6 +22,49 @@
     "1407215334327603200",
     "1407523834069086208",
     "1407884276381609989",
-    "1407979501754011654"
```
Comment (theyokohamalife) [inline data/testimonials.json:25]: Just missing a comma

Comment (renderf0x) [reply data/testimonials.json:25]: Nah, this was the previous last line and it's been replaced by itself with a comma - see the additions. I always make sure to validate json files before committing them =)

Comment (renderf0x) [reply data/testimonials.json:25]: @theyokohamalife can you approve? Let me know if there are any actual changes you want.

**Review by theyokohamalife — APPROVED** [review]
LGTM!

**Pull request merged**